mantle/cmd/kola/testiso.go

```diff
@@ -168,17 +168,23 @@ func newQemuBuilder(outdir string) (*platform.QemuBuilder, *conf.Conf, error) {
 	//TBD: see if we can remove this and just use AddDisk and inject bootindex during startup
 	if system.RpmArch() == "s390x" || system.RpmArch() == "aarch64" {
 		// s390x and aarch64 need to use bootindex as they don't support boot once
-		builder.AddDisk(&platform.Disk{
+		err := builder.AddDisk(&platform.Disk{
 			Size: "12G", // Arbitrary
 
 			SectorSize: sectorSize,
 		})
+		if err != nil {
+			panic(err)
```

Comment:
Why do we `panic` here (and below) instead of returning the error to the caller?

Reply:
Make sense. Thank you for pointing it out. Fixed.